### ISSUE TITLE
g910 orion spectrum "|" and "\" keys

### ISSUE THREAD

**Issue by Draftex** [antonpup/Aurora#794]
### What are you experiencing an issue with?

- [ ] Aurora
- [ ] Aurora supported game, (Fill in the game)
- [ ] Aurora Wrapper
- [ ] Aurora Profile
- [x] Aurora supported device, (Fill in problematic devices)
- [ ] Aurora Scripts
- [ ] Aurora Updater
- [ ] Aurora Lighting issue
- [ ] Other, (Fill in another issue here)

### What kind of an issue is it?

- [ ] Crash
- [ ] Performance problems
- [ ] Visual bug
- [ ] Localization issue
- [ ] Enhancement suggestion
- [x] Device issue
- [ ] Other, (Fill in another type of issue here)

### Expected behavior:
Explain here what should happen.
The 2 "\" keys (also "|" if you hold shift) should work with all profiles

### Actual behavior:
Explain here what happened instead.
The 2 "\" keys (also "|" if you hold shift) seem completely separated from
aurora, you can't see them in profiles or in the keyboard previews;
I believe the problem is that aurora isn't optimized for the new g910.

### How often does this problem occur?

This problem occurs... (every time)

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1. use the newer g910 orion spectrum keyboard from logitech
2. set any color (even no color) for the keyboard in logitech gaming software
3. use any profile on aurora

### What is the version of Aurora you are experiencing this issue with?

My version is... (beta v0.6.0b (but it never worked on any other version))

### Does the previous version of Aurora have this issue?
The previous version, (not truly sure, I had it then switched to the new one), did have this issue.

### Please include any logs, screenshots, or files that are related to this issue:

For logs, you can use: http://pastebin.com/
For screenshots, you can use: http://imgur.com/
For files, you can use: http://www.tinyupload.com/ , https://www.dropbox.com/ , or attach them to the issue.

**Comment by Draftex** [comment]
sorry I didn't leave any logs or images, I though you should have a pretty good understanding of what the issue was

**Comment by magnus97** [comment]
try updating to dev-3 since it works for me

**Comment by Draftex** [comment]
uninstalled aurora, downloaded dev-3, still doesn't work. thanks for replying quickly though!

**Comment by simon-wh** [comment]
What layout do you have for your keyboard and what one do you have selected in Aurora?

**Comment by Draftex** [comment]
If you by layout you mean "QWERTY", "QWERTZ" etc. I have a querty keyboard and i have selected United Sates in aurora (though my problem occurs with "automatic" aswell.

**Comment by Draftex** [comment]
Sorry, I meant I had a "QWERTY" keyboard, my mistake.

**Comment by Hottracks** [comment]
Agreed, this key is misrepresented in your keyboard layout inside Aurora as well. It should be above the "Enter" key, not next to it. This "\ - |" key is not being controlled by Aurora at all. It's not affected by any effect in the app.

**Comment by Draftex** [comment]
Well thanks for replying, I now just use the color cycle

**Comment by simon-wh** [comment]
@Draftex Could you find an image of the exact layout your keyboard has? I'm confused as to why you aren't able to find a layout in Aurora that works.
